clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StorePartitionDataReceiver.java

```diff
@@ -135,4 +135,9 @@ public String toString() {
     return this.getClass().getSimpleName() + "{" + "VT=" + storeIngestionTask.getVersionTopic() + ", topicPartition="
         + topicPartition + '}';
   }
+
+  // for testing purpose only
+  public int getKafkaClusterId() {
```

Comment:
nit: If it is for unit test then it can just be default modifier for least exposure.